data/json/vehicle_parts.json

```diff
@@ -37,6 +37,29 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "reclining_seat",
+        "name" : "reclining seat",
+        "symbol" : "#",
+        "color" : "red",
+        "broken_symbol" : "*",
+        "broken_color" : "red",
+        "damage_modifier" : 60,
+        "durability" : 100,
+        "item" : "seat",
+        "difficulty" : 3,
+        "size" : 25,
+        "location" : "center",
+        "flags" : [
+            "BED", "SEAT", "BOARDABLE", "BELTABLE", "CARGO"
```

Comment:
Many of the regular seats are installed in vehicles that definitely don't/can't have reclining seats. Trucks, vans with walls behind the seat, etc. I can tell you from experience that it's much more comfortable to sleep in a fully reclining seat than in the front seat of a truck :)